pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/argoproj.io/v1alpha1/Workflow/testdata/revisereplica-test.yaml

```diff
@@ -0,0 +1,102 @@
+# test case for revise replica
+# case1. Workflow: revise replica test (scale up)
+# case2. Workflow: revise replica test (no change)
+# case3. Workflow: revise replica test (scale down)
```

Comment:
Only one case is needed to cover the content being tested. I suggest keeping case 1 or case 3, but the description needs to be revised. The current description is somewhat inappropriate.